packages/jest-editor-support/src/Runner.js

```diff
@@ -177,10 +177,10 @@ export default class Runner extends EventEmitter {
 
   findMessageType(buf: Buffer): MessageType {
     const str = buf.toString('utf8', 0, 58);
-    if (str === 'No tests found related to files changed since last commit.') {
+    const lastCommitRegex = /No tests found related to files changed since ((last commit)|([a-z0-9]+))./;
```

Comment:
@SimenB I just realised that this regular expression should include the quotation marks. The reason why the tests passed was cause the tests were wrong. Going to make a new PR to fix this.